src/utilities/sql/SqlFile_Impl.cpp

```diff
@@ -447,16 +447,16 @@ namespace openstudio{
       // m_hasYear & m_hasIlluminanceMapYear are both default initialized to true
       if (version < VersionString(9, 2)) {
         m_hasIlluminanceMapYear = false;
+      }
 
-        if (version < VersionString(8, 9)) {
+      if (version < VersionString(8, 9)) {
+        m_hasYear = false;
+      } else {
+        // Check if zero
+        boost::optional<int> maxYear = execAndReturnFirstInt("SELECT MAX(Year) FROM Time");
+        if (!maxYear.is_initialized() || maxYear.get() <= 0) {
+          LOG(Warn, "Using EnergyPlusVersion version " << version.str() << " which should have 'Year' field, but it's always zero");
```

Comment:
Ok, that can't really hurt to be more careful here. 
Initially I added that becaus the 9.0 was problematic, others would work fine.